doma-spring-boot-autoconfigure/src/main/java/org/seasar/doma/boot/event/DomaEventEntityListener.java

```diff
@@ -0,0 +1,46 @@
+package org.seasar.doma.boot.event;
+
+import org.seasar.doma.jdbc.entity.*;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.context.ApplicationEventPublisherAware;
+
+public class DomaEventEntityListener<T> extends /* ugly hack */NullEntityListener<T>
```

Comment:
Is it necessary to extend the `NullEntityListener` class?
You can implement the `EntityListener` interface directly.